lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -201,6 +201,16 @@ def get_upload_status(app_id, upload_token)
         return UploadStatusResponse.new(response.body)
       end
 
+      def self.configure_connection(conn) # :nodoc:
+        conn.response(:json, parser_options: { symbolize_names: true })
+        conn.response(:raise_error) # raise_error middleware will run before the json middleware
+        # The Firebase API seems to use the POST verb where it should use PUT, and all of those
+        # are idempotent. Retry them since they can randomly fail in the wild.
```

Comment:
Hi Phillip, could you add a retry interval, interval randomness and backoff to add some delay and randomness to retries? https://github.com/lostisland/faraday/blob/master/lib/faraday/request/retry.rb#L14

Reply:
this should be good - 
```
    #                          interval: 0.05,
    #                          interval_randomness: 0.5,
    #                          backoff_factor: 2,
```

Reply:
Sure! I can't believe there aren't defaults. What a footgun. Seems like a good way to destroy a server.

I'm off work for the day, but can pick it back up tomorrow.

Reply:
Haha yeah was a bit surprised with no defaults too. Thank you!